prov/shm/src/smr_progress.c

```diff
@@ -345,9 +346,10 @@ static int smr_mmap_peer_copy(struct smr_ep *ep, struct smr_cmd *cmd,
 	return ret;
 }
 
-static int smr_progress_mmap(struct smr_cmd *cmd, struct iovec *iov,
-			     size_t iov_count, size_t *total_len,
-			     struct smr_ep *ep)
+static int smr_progress_mmap(struct smr_cmd *cmd, enum fi_hmem_iface iface,
+							 uint64_t device, struct iovec *iov,
```

Comment:
The indentation in this patch looks real funky. Please fix to align with the rest of the code - ie align function arguments with the parentheses.

Reply:
Oops, yeah my tab spacing is definitely off here. Fixing now...